pywbem/cim_operations.py

```diff
@@ -3795,11 +3824,12 @@ def ModifyInstance(self, ModifiedInstance, IncludeQualifiers=None,
             This parameter has been deprecated in :term:`DSP0200`. Clients
             cannot rely on it being implemented by WBEM servers.
 
-          PropertyList (:term:`py:iterable` of :term:`string`):
-            An iterable specifying the names of the properties to be
-            modified (case independent).
+          PropertyList (:term:`py:iterable` of :term:`string` or :term:`string`):
+            An iterable specifying the names of the properties (or a  string
+            that defines a single property) to be included in the returned
+            instances (case independent).
 
-            An empty iterable indicates to modify no properties.
+            An empty iterable indicates to include no properties.
```

Comment:
This paragraph and the previous paragraph should not talk about "properties to be included in the returned instances" but about "properties to be modified". See the original text.